omod-2.0/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs2_0/ConceptAttributeTypeController2_0Test.java

```diff
@@ -73,4 +74,20 @@ public void shouldCreateConceptAttributeType() throws Exception {
 
     }
 
+    @Test
+    public void shouldUpdateConceptAttributeType() throws Exception {
+        String CONCEPT_ATTRIBUTE_TYPE_UUID ="9516cc50-6f9f-11e0-8414-001e378eb67e";
```

Comment:
Do you change the value of `CONCEPT_ATTRIBUTE_TYPE_UUID` anywhere in this method?

Reply:
ohh I get it ....let me make it final